iocage_lib/iocage.py

```diff
@@ -1164,8 +1164,7 @@ def rename(self, new_name):
 
         new_mountpoint = f"{self.iocroot}/{_folders[0]}/{new_name}"
 
-        if (os.path.isdir(new_mountpoint) or
-                os.path.isdir(f"{self.iocroot}/{_folders[1]}/{new_name}")):
+        if ioc_common.match_to_dir(self.iocroot, new_name, old_mountpoint):
```

Comment:
Let's specify the optional argument here for old_mountpoint. In case the interface changes.